index.html

```diff
@@ -28,14 +28,16 @@ <h2 class="cta">Find events near you</h2>
       </div>
     </div>
   </div>
-  <div class="map">
-    <h2>Imagine a map</h2>
-    <div id="webpack-test">
-    </div>
-  </div>
+  <div class="map" id="map"></div>
+
+  <!-- First draw an empty map -->
+  <script src="initialize-map.js"></script>
+
+  <!-- Then grab data dependencies -->
+  <script src="https://s3.amazonaws.com/thirdbear-backups/aclu/public/action_events.js"></script>
```

Comment:
These could also be loaded in parallel _inside_ the main bundle before booting the app but I'm keeping it simple for now.